spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/statemachine/RollbackStartAction.java

```diff
@@ -54,9 +56,17 @@ public RollbackStartAction(ReleaseRepository releaseRepository) {
 
 	@Override
 	protected void executeInternal(StateContext<SkipperStates, SkipperEvents> context) {
+		log.debug("Starting to execute action");
```

Comment:
Could this be, `Starting to execute rollback action`

Reply:
Doesn't matter, logging will tell class where debug happens.